_posts/2024-01-18-How_to_run_Metal3_website_locally_with_Jekyll.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "How to run Metal3 website locally with Jekyll"
+date: 2024-01-18
+draft: false
+categories:
+  ["metal3", "baremetal", "metal3-dev-env", "documentation", "development"]
+author: Salima Rabiu
+---
+
+## Introduction
+
+If you’re a developer or contributor to the Metal3 project, you may need to run the Metal3
+website locally to test changes and ensure everything looks as expected before deploying
+them. In this guide, we’ll walk you through the process of setting up and running Metal3’s
+website locally on your machine using Jekyll.
+
+## Prerequisites
+
+Before we begin, make sure you have the following prerequisites installed on your system:
+
+- Ruby: Jekyll, the static site generator used by Metal3, is built with Ruby. Install Ruby
+  and its development tools by running the following command in your terminal:
+
+```bash
+sudo apt install ruby-full
+```
```

Comment:
Could you indent this so it becomes part of the list item?
````suggestion
  ```bash
  sudo apt install ruby-full
  ```
````